code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/state/AepCardUIState.kt

```diff
@@ -12,12 +12,16 @@
 package com.adobe.marketing.mobile.aepcomposeui.aepui.state
 
 /**
- * Interface representing the state of an AEP card.
+ * Class representing the state of an AEP card.
  *
- * This interface includes common properties like `dismissed` and `selected`,
- * which can be shared across different card states.
+ * This class includes the common properties `dismissed`, `selected`, and `read` which are common across different card states.
+ *
+ * @property dismissed Indicates whether the card has been dismissed.
+ * @property selected Indicates whether the card is selected.
+ * @property read Indicates whether the card has been read.
  */
-interface AepCardUIState {
-    val dismissed: Boolean
-    val selected: Boolean
-}
+open class AepCardUIState(
+    open var dismissed: Boolean = false,
+    open var selected: Boolean = false,
+    open var read: Boolean = false
```

Comment:
Can we make these `val` so that the state is fixed and can only be updated via updateState() with using state.copy(..).